test/system/feedbacks_test.rb

```diff
@@ -0,0 +1,189 @@
+require 'application_system_test_case'
+
+class FeedbacksTest < ApplicationSystemTestCase
+  def setup
+    @user_volunteer = create :user_volunteer, email: 'volunteer@example.com'
+    @volunteer = create :volunteer, user: @user_volunteer
+    @assignment = create :assignment, volunteer: @volunteer
+    @superadmin = create :user
+    @other_volunteer = create :volunteer, user: create(:user_volunteer)
+    @group_offer = create :group_offer, necessary_volunteers: 2, title: 'some_group_offer',
+      volunteers: [@volunteer, @other_volunteer]
+  end
+
+  def setup_feedbacks
+    create :feedback, feedbackable: @assignment,
+    volunteer: @volunteer, author: @superadmin, comments: 'author superadmin'
+    create :feedback, feedbackable: @group_offer, author: @user_volunteer,
+    volunteer: @volunteer, comments: 'some_group_offer_feedback'
+    create :feedback, feedbackable: @group_offer, author: @superadmin, volunteer: @volunteer,
+      comments: 'some_group_offer_superadmin_feedback'
+    create :feedback, volunteer: @other_volunteer, feedbackable: @group_offer,
+      author: @superadmin, comments: 'other_volunteers_group_offer_feedback'
+  end
+
+  test 'volunteer can see assignment feedbacks index' do
+    setup_feedbacks
+    login_as @user_volunteer
+    visit root_url
+    click_link 'volunteer@example.com'
+    click_link 'Show profile'
+    within '.assignments-table' do
+      click_link 'Feedback index'
+    end
+    refute page.has_text? 'author superadmin'
```

Comment:
Maybe here you could also check if a text from the @user_volunteer exists, so as to check that it can indeed visit that page.